dist/freeciv-modpack.appdata.xml.in

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
-    <id type="desktop">net.longturn.freeciv_modpack_qt</id>
+    <id type="desktop">net.longturn.freeciv21.modpack_qt</id>
```

Comment:
I think this should be freeciv21_modpack_qt

Reply:
Is there somewhere in the spec/a guideline that says so? I prefer the dot because it introduces a hierarchy (longturn.net isn't just fc21).

Reply:
If you want to create a another level for hierarchy then a period makes sense there. If you don't then use the underscore. That is how I read the spec.

Reply:
If we are getting rid of qt since everything is qt - does it make sense to change `modpack_qt` to either `modpack` or `modpack_installer`?